docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-node.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Build a Node.js application that uses YSQL
+headerTitle: Build a Node.js application
+description: Build a simple Node.js application using the driver and using the YSQL API to connect to and interact with a Yugabyte Cloud cluster.
+menu:
+  latest:
+    parent: cloud-build-apps
+    name: Node.js
+    identifier: cloud-node
+    weight: 400
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+The following tutorial shows a small [Node.js application](https://github.com/yugabyte/yugabyte-simple-node-app) that connects to a YugabyteDB cluster using the [node-postgres module](../../../../reference/drivers/ysql-client-drivers/#node-postgres) and performs basic SQL operations. Use the application as a template to get started with Yugabyte Cloud in Node.js.
```

Comment:
You can also link this Build a Node.js page in this reference section : https://docs.yugabyte.com/latest/reference/drivers/ysql-client-drivers/#node-postgres

Reply:
Haven't yet decided what will happen with the older examples, they may be replaced by this one - which works equally well for local install